meta-ibm/recipes-phosphor/webui/webui-vue_%.bbappend

```diff
@@ -23,5 +23,5 @@ do_install:append:p10bmc() {
     install -m 0755 ${WORKDIR}/inspur-login-logo.svg.gz ${D}${datadir}/www/img
     install -m 0755 ${WORKDIR}/inspur-logo-header.svg.gz ${D}${datadir}/www/img
     install -m 0755 ${WORKDIR}/blankLogo.svg.gz ${D}${datadir}/www/img
-    install -m 0755 ${WORKDIR}/changeLogo.sh ${D}${datadir}/www
+    install -m 0755 ${WORKDIR}/changeLogo.sh ${D}${bindir}
```

Comment:
Need to create the dir to fix the compile error that Andrew mentioned:

```
+    install -d ${D}${bindir}
+    install -m 0755 ${WORKDIR}/changeLogo.sh ${D}${bindir}
```

After building, verify that the change-logo.service file has the expected line change, location:
```
tmp/work/p10bmc-openbmc-linux-gnueabi/obmc-phosphor-image/1.0-r0/rootfs/lib/systemd/system/change-logo.service
```

And that the script exists under this path:
```
tmp/work/p10bmc-openbmc-linux-gnueabi/obmc-phosphor-image/1.0-r0/rootfs/usr/bin/changeLogo.sh
```

Probably it'd be good too to flash the image to a system to verify the logo script runs as expected.

Document your results in the commit message under the "Tested:" tag as Andrew requested.

Reply:
Thanks @anoo1 @geissonator 
Updated